l10n_it_fatturapa_out/models/partner.py

```diff
@@ -16,7 +16,11 @@ class ResPartner(models.Model):
 
     @api.constrains("max_invoice_in_xml")
     def _validate_max_invoice_in_xml(self):
-        if self.max_invoice_in_xml < 0:
-            raise ValidationError(
-                _("The max number of invoice to group can't be negative")
-            )
+        for partner in self:
+            if partner.max_invoice_in_xml < 0:
+                raise ValidationError(
+                    _(
+                        "The max number of invoice to group can't be negative with partner %s",
+                        partner.name,
```

Comment:
`partner.name`  non è da tradurre: va fuori dalle parentesi

Reply:
@tafaRU le parentesi mi sembrano corrette, viene tradotta solo la stringa che si trova tra i doppi apici "".

Reply:
Lo stile "nuovo" è documentato e consigliato qui:
https://www.odoo.com/documentation/16.0/developer/howtos/translations.html?highlight=translation#variables

Vedi anche [qui](https://github.com/OCA/l10n-italy/pull/2907#discussion_r1192379539).
Avevo anche fatto delle prove in locale e funziona egregiamente, se nella stringa tradotta viene dimenticato il `%s` non va in errore ma ripiega sulla stringa in inglese.

Reply:
Ciao, 

sinceramente sono andato a guardare il codice standard di Odoo 16 e ho preso riferimento da lì.
Se cercate ValidationError ci sono una serie infinita di righe di codice che usano lo %s all'interno della parentesi

Aiutatemi a capire dove sto sbagliando... :no_mouth:

Reply:
@tafaRU 
Vedi anche https://github.com/odoo/odoo/blob/14.0/odoo/tools/translate.py#L446-L457

Reply:
Grazie @primes2h, non lo conoscevo :+1: